docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md

```diff
@@ -9,8 +9,10 @@ Besu nodes can operate (that is, submit transactions and receive blocks) in a Qu
 
 To connect to a Quorum IBFT 1.0 network:
 
-1. In the Quorum IBFT 1.0 [genesis file](../Genesis-File.md), update the consensus protocol specified in the `config`
-   property from `istanbul` to `ibft`.
+1. In the Quorum IBFT 1.0 [genesis file](https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file):
+
+    - Update the consensus protocol specified in the `config` property from `istanbul` to `ibft`.
```

Comment:
There are further changes required unfortunately going from GoQuorum to Besu
* GoQuorum istanbul config property epoch becomes epochlength in Besu ibft config
* GoQuorum --istanbul.blockperiod cmd line option becomes the blockperiodseconds in Besu ibft config
* GoQuorum --istanbul.requesttimeout cmd line option needs to be converted from milliseconds to seconds and becomes requesttimeoutseconds in Besu ibft config
* Besu doesn't support sticky validator nodes. So a GoQuorum config with a policy of 1 cannot be used with Besu

Reply:
Added these changes. Can you re-review @jframe?